jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -37,6 +25,18 @@ class Markers(PluginTemplateMixin, ViewerSelectMixin, TableMixin):
     """
     template_file = __file__, "markers.vue"
 
+    _default_table_values = {'spectral_axis': np.nan,
+                             'spectral_axis:unit': '',
+                             'slice': np.nan,
+                             'pixel': (np.nan, np.nan),
+                             'pixel:unreliable': None,
+                             'world': (np.nan, np.nan),
+                             'world:unreliable': None,
+                             'value': np.nan,
+                             'value:unit': '',
+                             'value:unreliable': None,
+                             'index': np.nan}

```

Comment:
by moving this in the class, it can be overridden by a plugin that inherits from jdaviz `Markers`